### DISCUSSION TITLE
godaddy API key 更新域名无法成功

### DISCUSSION THREAD

**Discussion by wuyz320** [jeessy2/ddns-go#866] [Q&A]
通过设置godaddy 的 API key的方式新增或更新域名无法成功。
报错日志是这样的：
2023/09/25 09:15:01 请求接口https://api.godaddy.com/v1/domains/xxx.com/records/A/ndp31失败! ERROR: Put "https://api.godaddy.com/v1/domains/xxx.com/records/A/ndp31": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
2023/09/25 09:15:01 更新域名解析 ndp31.xxx.com 失败！

**Comment by jeessy2** [comment]
检查你网络，国内是用不起的
